test-cases/artifacts/docker.yaml

```diff
@@ -6,7 +6,8 @@ n_loaders: 0
 n_monitor_nodes: 0
 nemesis_class_name: 'NoOpMonkey'
 scylla_linux_distro: 'centos'
-scylla_version: '3.3.rc1'
+scylla_version: '3.3.rc2'
```

Comment:
Can we not hardcode those values ? It can lead to mistakes. 
Like pulling the wrong debug symbols.

Reply:
It's more like example. This parameter will be overridden by Jenkins job.